pixels/endpoints/general.py

```diff
@@ -0,0 +1,175 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException, Request, Response
+
+from pixels.constants import Ratelimits, Sizes
+from pixels.models import GetSize, Message, Pixel
+from pixels.utils import auth, ratelimits
+
+log = logging.getLogger(__name__)
+router = APIRouter(tags=["Canvas Endpoints"], dependencies=[Depends(auth.JWTBearer())])
+
+
+@router.get("/size", response_model=GetSize)
+async def size(request: Request) -> GetSize:
+    """
+    Get the size of the pixels canvas.
```

Comment:
```suggestion
    Get the size of the canvas.
```
Or if we want to keep it, capitalize it (since it's the Pixels project, not pixels as the word)
```suggestion
    Get the size of the Pixels canvas.
```